### ISSUE TITLE
[bug] nametags don't work without color code

### ISSUE THREAD

**Issue by RalphORama** [WiIIiam278/Velocitab#112]
I had player nametags stop working after some configuration changes. After a bit of fiddling, I discovered the issue: it appears using just `%nametag%` or `&r%nametag%` makes nametags invisible, but `&f%nametag%` makes them visible.

I originally set my config to look like this because I didn't want VelociTab changing nametags:

```yaml
nametags:
  default: '%username%'
```

I tried adding `&r` to see if that would make nametags visible, but that did not work:

```yaml
nametags:
  default: '&r%username%'
```

Adding `&f` finally fixed the issue:

```yaml
nametags:
  default: '&f%username%'
```

**Comment by WiIIiam278** [comment]
we actually do handle not setting a color here:

https://github.com/WiIIiam278/Velocitab/blob/7005ceccd7f2dbde41dc34ea263fea0353de8a5b/src/main/java/net/william278/velocitab/packet/UpdateTeamsPacket.java#L96C30-L96C30

but I think something might be messing up here before that point:

https://github.com/WiIIiam278/Velocitab/blob/7005ceccd7f2dbde41dc34ea263fea0353de8a5b/src/main/java/net/william278/velocitab/packet/UpdateTeamsPacket.java#L96C30-L96C30

I also note we store nametags per _role_, which I think will probably break if player-specific placeholders are used (should prob be per-user? maybe not though).

**Comment by RalphORama** [comment]
Curious. In my experience setting `[&r]%username%` and running `velocitab reload` made usernames disappear, then setting `&f%username%` and reloading made them come back.

**Comment by alexdev03** [comment]
If you want the default nametags just remove the default value and make the section empty. nametags: {}
If you use the latest version this problem is fixed without changing the section, you only need to recreate the config. I added an option to allow this feature or not.

**Comment by alexdev03** [comment]
> we actually do handle not setting a color here:
> 
> https://github.com/WiIIiam278/Velocitab/blob/7005ceccd7f2dbde41dc34ea263fea0353de8a5b/src/main/java/net/william278/velocitab/packet/UpdateTeamsPacket.java#L127-L133
> 
> but I think something might be messing up here before that point:
> 
> https://github.com/WiIIiam278/Velocitab/blob/7005ceccd7f2dbde41dc34ea263fea0353de8a5b/src/main/java/net/william278/velocitab/packet/ScoreboardManager.java#L148
> 
> I also note we store nametags per _role_, which I think will probably break if player-specific placeholders are used (should prob be per-user? maybe not though).

Each user is assigned a specific and unique role, which mitigates any potential issues.

**Comment by alexdev03** [comment]
news @RalphORama ?

**Comment by WiIIiam278** [comment]
Could you try with the latest Alpha version? https://modrinth.com/plugin/velocitab/version/

**Comment by WiIIiam278** [comment]
closing as this should have been fixed in the latest alpha via #108
